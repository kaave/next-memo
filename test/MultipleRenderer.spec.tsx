import React from 'react';
import { act as domAct } from 'react-dom/test-utils';
import { act as testAct, create, ReactTestRenderer } from 'react-test-renderer';

const App = () => <div>App</div>;

let root: ReactTestRenderer;
domAct(() => {
  testAct(() => {
    root = create(<App />);
  });
});

it('Multiple renderer test', () => {
  expect(root).toMatchSnapshot();
});
