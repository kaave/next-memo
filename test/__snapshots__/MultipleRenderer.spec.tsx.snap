// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` 1`] = `
<div>
  App
</div>
`;

exports[`Multiple renderer test 1`] = `
<div>
  App
</div>
`;
